WordPress/Classes/Utility/BuildInformation/Build.swift

```diff
@@ -0,0 +1,73 @@
+@objc
+class Build: NSObject {
+    @objc
+    enum BuildConfiguration: Int {
+        /// Development build, usually run from Xcode
+        case localDeveloper
+        /// Continuous integration builds for Automattic employees to test branches & PRs
+        case a8cBranchTest
+        /// Beta released internally for Automattic employees
+        case a8cPrereleaseTesting
+        /// Production build released in the app store
+        case appStore
+    }
+
+    /// Returns true if any of the given build types matches the current build
+    ///
+    /// Example:
+    ///
+    ///     let enableExperimentalStuff = build(.localDeveloper, .a8cBranchTest)
+    ///
+    static func `is`(_ testConfiguration: BuildConfiguration) -> Bool {
```

Comment:
The backticks to name this `is` make me ponder, but I think fundamentally if this were a Swift enum instead of a class—if we don't need to use this in ObjC, as I suspect we may not—we could use built-in equality to evaluate this instead of providing a method?

Reply:
Kind of related to what I said above.  My take on this is that as long as we have Obj-C code and tests, making these methods and classes Swift-only would be not ideal.

Reply:
I understand the idea that _maybe_ we'll do new development with Objective-C, but atm our principle is that all new code is Swift, making the chance that we have to support ObjC pretty slim. fwiw, I vote we do this the nice Swifty way 😜